users/vieting/experiments/switchboard/hybrid/feat/baseline_args.py

```diff
@@ -96,7 +101,7 @@ def get_nn_args(nn_base_args, num_epochs, evaluation_epochs=None, prefix=""):
 
 def get_nn_args_single(
     num_outputs: int = 9001, num_epochs: int = 500, evaluation_epochs: Optional[List[int]] = None,
-    peak_lr=1e-3, feature_args=None, returnn_args=None,
+    peak_lr=1e-3, feature_args=None, returnn_args=None,specaug_mask_sorting=False,
```

Comment:
It would be sufficient to add `specaug_mask_sorting` to `returnn_args` and only introduce this as input arg in `get_returnn_config()`.

Reply:
Btw, there is a space missing in this line. I wonder why black does not complain about it?

Reply:
Ah, the black test does not run on files under `users/`.